.changeset/five-lobsters-juggle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/combobox': patch
+'@twilio-paste/core': patch
+---
+
+[Combobox]: Add suffix to Chevron Icon wrapper. This ensures that the element string value for the combobox's chevron wrapper will always include the suffix "_CHEVRON_WRAPPER", and use either the default element name, or the consumer provided custom element name. No consumer change required.
```

Comment:
@SiTaggart I just updated this one, because I think the previous change log wasn't super descriptive of the actual change.